sources/packages/api/src-sql/MSFAANumbers/Add-col-offering-cancelled.sql

```diff
@@ -0,0 +1,27 @@
+-- Create offering_intensity
+ALTER TABLE
+    sims.msfaa_numbers
+ADD
+    COLUMN IF NOT EXISTS offering_intensity VARCHAR(10) NOT NULL DEFAULT 'Full Time';
+
+COMMENT ON COLUMN sims.msfaa_numbers.offering_intensity IS 'Offering intensity for the MSFAA Number Part Time/ Full Time.';
+
+ALTER TABLE
+    sims.msfaa_numbers
+ADD
+    COLUMN IF NOT EXISTS new_issuing_province VARCHAR(2);
```

Comment:
I guess `new_issuing_province` will always be 2 characters, if so, pls use 'CHAR`. 

Also for province too we can use enum or a table. thoughts @dheepak-aot  @abschwenker  @guru-aot

Reply:
If we know the values of the province and how they come in the file, its okay to create an enum, else it will error out if we are not sure while processing the file. Or it should be good with CHAR(2) as you said @ann-aot .

Reply:
the file layouts only support 2 characters. But we wont be using these codes anywhere as its only for records. For this likely not worth using an ENUM type which if there is an issue will cause an issue with the insert